apps/tasks/views.py

```diff
@@ -100,3 +113,74 @@ def perform_create(self, serializer):
         task_id = self.request.data['task']
         task = get_object_or_404(Task, id=task_id)
         send_notification([task.owner.email], "New comment!", "You task was commented")
+
+
+class TimerViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated]
```

Comment:
mai bine de pus in settings sa fie default permision_class = IsAuthenticated, si deja unde nu trebuie el de schimbat